package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/button/react-simple-scroll#readme",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
+    "react": "^0.14.0 || ^16.0.0",
```

Comment:
Seems like we're dropping support for the `^0.14.0` line as well, correct?

Reply:
@griffinmyers The biggest change that we'll be applying here is using `prop-types` package instead of `React.PropTypes`, and as far as I can tell, it `prop-types` supports `0.14` and we're not implementing any validator functions in our current tests: https://github.com/facebook/prop-types#compatibility. 

Everything else in this package seems ok React-wise. We're not using any soon-to-be deprecated lifecycle hooks so this component should work in this range and after installing React 16 and running tests using enzyme's React 16 adapter everything is ✅

That being said, we can be a _little_ more strict and maybe range it between `15` and `16` to ensure that people should be using more recent versions of React when using `2.0` of this component. LMK what you think!

Reply:
Generally I think it's nice for libraries to be as permissive as is reasonable--but also think it's reasonable for us to take this opportunity to set a new lower-bound on support, especially given folks can continue to use `1.0.0` in perpetuity (this isn't exactly under active development).

I'm happy as long as we're compatible with any versions we say we are, and the range of compatible versions is sensible (i.e. there aren't gaps in it). The pragmatic choice is probably just declaring 16.0.0 as the new lowest support version.

Reply:
Gotcha, makes sense. Declaring `16.0.0` as the next iteration of this component is probably the best choice since, people can still use `1.0.0` if they are using older version of React. I'll update this PR 👍